### DISCUSSION TITLE
Minimum voltage triggered while discharging Prada2013

### DISCUSSION THREAD

**Discussion by srdallas1** [pybamm-team/PyBaMM#4481] [Q&A] [answered]
Hi all, when I try to fully discharge the LFP cell defined in Prada2013 (discharging at a c-rate of 1) I encounter a problem - "'event: Minimum voltage [V]' was triggered", which occurs before reaching 0% SOC. Currently, I can get around the problem by using a termination criteria based off the 'Discharge capacity [A.h]' and the initial SOC and not allowing the 'SOC' to drop too low. 

After I had a look through the discussions, the code and the documentation, I concluded it might be due to the actual capacity of the cells being less than the nominal cell capacity (which from what I understand is used to set the discharge current when discharging). For example, I noticed that the minimum voltage event occurs when I discharge to less than around 15% SOC, which makes me think that the actual capacity of the cells is closer to 85% of the nominal cell capacity of 2.3Ah. However, when I tried to calculte the real value of the capacity, using the method here https://docs.pybamm.org/en/stable/source/examples/notebooks/models/electrode-state-of-health.html, I found that it is very close to 2.3 Ah. 

Any guidance on where I might be going wrong would be very much appreciated.

**Comment by DrSOKane** [comment]
Try using `plot_voltage_components()` to see if there are any overpotentials causing the voltage limit to be hit before the cell is fully discharged. See the example notebook [here](https://docs.pybamm.org/en/latest/source/examples/notebooks/plotting/plot-voltage-components.html).

**Reply by srdallas1** [reply]
Thanks @DrSOKane, I can see from the plotting the voltage components that the culprit is the negative particle concentration overpotential:

![image](https://github.com/user-attachments/assets/04315742-3fe1-4b51-b6c4-ce36a5ba3b25)

If I lower the c-rate, I am able to discharge the battery more before the minimum voltage event is triggered. From doing a bit of reading, it seems like the problem might be that the battery is not designed to discharge at such a high c-rate, which is causing the large value of negative particle concentration overpotential towards the end of the discharging. 

I would like to be able to discharge at a c-rate of up to 3. Are there any realisitic work arounds you know of to do this or is it simply that I need to change the parameter set I'm using? 

My main focus is the lumped thermal effects of the charging/discharging on the LFP cell, which I obtain from running an SPMe model with the Prada2013 parameter set. If you know of another way to approach this, I would appreciate any tips.

Thanks again!

**Reply by DrSOKane** [reply, marked as answer]
The parameter `"Negative particle diffusivity [m2.s-1]"` is set to 3E-15 for the `Prada2013` cell, which is probably too small. Try increasing it, or making it a function of concentration.
